man/rc-update.8

```diff
@@ -35,7 +35,7 @@
 OpenRC uses named runlevels.  Rather than editing some obscure
 file or managing a directory of symlinks,
 .Nm
-exists to quickly add or delete services to and from from different runlevels.
+exists to quickly add or delete services to and from different runlevels.
```

Comment:
Spell checkers have been flagging duplicate words for many decades, but I've only recently started to work with them. It's possible I've messed up in places, but I don't thinks so.